asciidoc/courses/drivers-java/modules/1-driver/lessons/4-object-mapping/lesson.adoc

```diff
@@ -0,0 +1,139 @@
+= Mapping Results to Java Objects
+:type: lesson 
+:slides: true
+:minutes: 15
+:order: 5
+
+[.slide.col-2]
+== Introduction
+
+[.col]
+As of the Neo4j Java driver 5.28.5, the driver includes an link:https://github.com/neo4j/neo4j-java-driver/discussions/1640[object mapping feature] that allows you to map query results directly to Java objects. This feature simplifies the process of working with Neo4j data in Java applications by eliminating the need for manual mapping with raw data types.
+
+[.col]
+====
+[NOTE]
+.Object Graph Mapping (OGM)
+=====
+The driver's new object mapping is not a full-fledged OGM solution. For a more comprehensive option, check out the https://neo4j.com/docs/ogm-manual/current/[Neo4j OGM library^].
+=====
+====
+
+[.slide]
+== Domain Model
+
+For a person domain class, create a new file in `src/main/java/com/neo4j/app` and name it `Person.java`. This example will use Java records (rather than class), but classes works similarly.
```

Comment:
```suggestion
Nodes are represented by domain classes.

For the `Person` domain class, create a new file in `src/main/java/com/neo4j/app` and name it `Person.java`. 

[NOTE]
This example will use Java records (rather than class), but classes work similarly.
```